### PR TITLE
style: Mark equations where black formatting should not apply

### PR DESCRIPTION
**Pull request by ConnorStoneAstro**
Black formatting is intended for OOP python, not mathematical expressions. Often the way it breaks up an equation makes it harder to read. Here we mark equations that shouldn't be formatted by Black.

### CODE REVIEW

**Comment by lsetiawan** [comment]
Let's wait until https://github.com/Ciela-Institute/caustics/pull/124 is merged to merge this one.

**Review by lsetiawan — APPROVED** [review]
LGTM 🎉

File: src/caustics/cosmology.py [hunk 1]
```diff
@@ -420,14 +420,7 @@ def comoving_distance(
...
         ratio = (Om0 / Ode0) ** (1 / 3)
-        return (
-            self.hubble_distance(h0)
-            * (
-                self._comoving_distance_helper((1 + z) * ratio, params)
-                - self._comoving_distance_helper(ratio, params)
-            )
-            / (Om0 ** (1 / 3) * Ode0 ** (1 / 6))
-        )
+        return self.hubble_distance(h0) * (self._comoving_distance_helper((1 + z) * ratio, params) - self._comoving_distance_helper(ratio, params)) / (Om0 ** (1 / 3) * Ode0 ** (1 / 6))  # fmt: skip
```
Comment (AlexandreAdam) [inline src/caustics/cosmology.py:423]: In order to make this line more readable, I would rdefine new variables for the comoving distance primitive calculation above:

```
Ode0 = 1 - Om0
ratio = (Om0 / Ode0) ** (1 / 3)
H0 = self.hubble_distance(h0)
# fmt: off
D1 = self._comoving_distance_helper((1 + z) * ratio, params) # comoving distance primitive at z
D0 = self._comoving_distance_helper(ratio, params) # comoving distance primitive at z=0
# fmt: on
return H0 * (D1 - D0) / (Om0 ** (1 / 3) * Ode0 ** (1 / 6))  # fmt: skip
```

File: src/caustics/lenses/pseudo_jaffe.py [hunk 2]
```diff
@@ -149,21 +142,7 @@ def mass_enclosed_2d(
...
-            * (d_l * arcsec_to_rad) ** 2
-            / (scale_radius - core_radius)
-            * (
-                (core_radius**2 + theta**2).sqrt()
-                - core_radius
-                - (scale_radius**2 + theta**2).sqrt()
-                + scale_radius
-            )
-        )
+        return (2 * pi * surface_density_0 * core_radius * scale_radius * (d_l * arcsec_to_rad) ** 2 * ((core_radius**2 + theta**2).sqrt() - core_radius - (scale_radius**2 + theta**2).sqrt() + scale_radius) / (scale_radius - core_radius))  # fmt: skip
```
Comment (AlexandreAdam) [inline src/caustics/lenses/pseudo_jaffe.py:145]: I think this equation is a bit too long and is confusing me. 

Can we reformat it in a way that clarifies what is being computed and the units?

Below, I am attempting to do this although I am note sure this is right. 

```
        theta = theta + self.s # arcsec
        D_A = self.cosmology.angular_diameter_distance(z_l, params)  # Mpc
        Sigma_crit = self.cosmology.critical_surface_density(z_l, z_s, params)  # M_sun / Mpc^2
        Sigma_0 = self.get_convergence_0(z_s, params) * Sigma_crit  # M_sun / Mpc^ 2
        xi = D_A * theta * arcsec_to_rad # distance perpendicular to the line-of-sight (Mpc)
        mass = 2 * pi * Sigma_0 * D_A**2 * (core_radius * scale_radius * arcsec_to_rad)**2
        ratio = (core_radius**2 + theta**2).sqrt() - core_radius - (scale_radius**2 + theta**2).sqrt() + scale_radius) / (scale_radius - core_radius) #  no units
# The rest of the equation in terms of variables above
```

File: src/caustics/lenses/pseudo_jaffe.py [hunk 3]
```diff
@@ -294,20 +257,8 @@ def potential(
...
-        )
-        return coeff * (
-            (scale_radius**2 + R_squared).sqrt()
-            - (core_radius**2 + R_squared).sqrt()
-            + core_radius * (core_radius + (core_radius**2 + R_squared).sqrt()).log()
-            - scale_radius
-            * (scale_radius + (scale_radius**2 + R_squared).sqrt()).log()
-        )
+        coeff = -2 * self.get_convergence_0(z_s, params) * core_radius * scale_radius / (scale_radius - core_radius)  # fmt: skip
+        return coeff * ((scale_radius**2 + R_squared).sqrt() - (core_radius**2 + R_squared).sqrt() + core_radius * (core_radius + (core_radius**2 + R_squared).sqrt()).log() - scale_radius * (scale_radius + (scale_radius**2 + R_squared).sqrt()).log())  # fmt: skip
```
Comment (AlexandreAdam) [inline src/caustics/lenses/pseudo_jaffe.py:261]: Again this equation is too long, we should simplify it in terms of variables with proper units.

**Review by AlexandreAdam — CHANGES_REQUESTED** [review]
I've flagged 3 equations which I think are too long and deserve to be simplified in the code. The rest looks good to me.

File: src/caustics/lenses/pseudo_jaffe.py (hunk 3 above)
Comment (ConnorStoneAstro) [reply src/caustics/lenses/pseudo_jaffe.py]: I think I've split this one out. Though now it has been revealed that there are units of Mpc for the potential, so I've added a TODO here

**Pull request merged**